app/views/movies/search.html.erb

```diff
@@ -8,6 +8,6 @@
 
 <ul>
   <% @movies.each do |movie| %>
-  <li> <%= link_to movie[:title], "/movies/#{movie[:id]}" %>, Vote Average: <%= movie[:vote_average] %> </li>
+  <li> <%= link_to movie.title, "/movies/#{movie.id}" %>, Vote Average: <%= movie.vote_average %> </li>
```

Comment:
Smart use of facades and poros to make movie objects which have methods we can use in the views.